composables/massmint/useMassMint.ts

```diff
@@ -34,48 +34,41 @@ export const statusClass = (status?: Status) => {
 }
 
 export const useCollectionForMint = () => {
-  const collectionsEntites = ref<MintedCollection[]>()
+  const collections = ref<MintedCollection[]>()
   const { accountId } = useAuth()
   const { urlPrefix } = usePrefix()
 
   const { data, isPending } = useQuery({
     queryKey: ['collections-for-mint', accountId, urlPrefix],
-    queryFn: async () =>
-      accountId.value
-        ? (await useAsyncGraphql({
-            query: 'collectionForMint',
-            variables: {
-              account: accountId.value,
-            },
-          })).data.value
-        : null,
+    queryFn: async () => {
+      if (!accountId.value) {
+        return null
+      }
+
+      collections.value = []
+
+      return (await useAsyncGraphql({
+        query: 'collectionForMint',
+        variables: {
+          account: accountId.value,
+        },
+      })).data.value
+    },
   })
 
-  watch(data, () => {
+  watch(data, async () => {
     const collectionEntities = data.value?.collectionEntities
 
     if (collectionEntities?.length) {
-      const collections = collectionEntities
-        .map(collection => ({
-          ...collection,
-          lastIndexUsed: Math.max(
-            ...collection.nfts.map(nft => Number(nft.index)),
-          ),
-
-          alreadyMinted: collection.nfts?.length,
-          totalCount: collection.nfts?.filter(nft => !nft.burned).length,
```

Comment:
Will they be used somewhere?

Reply:
renamed the fields inside `collectionForMint.graphql`, so no need to map it again

![CleanShot 2024-10-28 at 07 48 03@2x](https://github.com/user-attachments/assets/88a9e5b3-7bfe-4d08-b6b8-7b508af8e65a)